GoogleForCreators/web-stories-wp
# Disable saving and previewing while uploading

## Summary

More info https://github.com/google/web-stories-wp/issues/2547

## Relevant Technical Choices

Had to move `AutoSaveHandler` below `MediaProvider` - checking the implications.

## To-do



## User-facing changes

The user shouldn't be able to save/autosave the story while something is uploading (Autosave/Save/Preview buttons are disabled).

## Testing Instructions

Try to save/auto the story (open Preview tab and check if the story is updated there) while something is uploading - you shouldn't be able to do it.

---



Fixes #2547
